### ISSUE TITLE
pbs NOVA videos fail 

### ISSUE THREAD

**Issue by scottstensland** [rg3/youtube-dl#3602]
youtube-dl -c -l  --verbose  http://www.pbs.org/wgbh/nova/earth/earth-from-space.html

[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-c', '-l', '--verbose', 'http://www.pbs.org/wgbh/nova/earth/earth-from-space.html']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2014.08.27.1
[debug] Python version 2.7.6 - Linux-3.13.0-34-generic-x86_64-with-Ubuntu-14.04-trusty
[debug] Proxy map: {}
[PBS] earth-from-space.html: Downloading webpage
ERROR: Unable to extract player URL; please report this issue on https://yt-dl.org/bug . Be sure to call youtube-dl with the --verbose flag and include its complete output. Make sure you are using the latest version; type  youtube-dl -U  to update.
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 521, in extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 176, in extract
    return self._real_extract(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/pbs.py", line 110, in _real_extract
    video_id, display_id = self._extract_ids(url)
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/pbs.py", line 91, in _extract_ids
    webpage, 'player URL')
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 397, in _search_regex
    raise RegexNotFoundError(u'Unable to extract %s' % _name)
RegexNotFoundError: Unable to extract player URL; please report this issue on https://yt-dl.org/bug . Be sure to call youtube-dl with the --verbose flag and include its complete output. Make sure you are using the latest version; type  youtube-dl -U  to update.

**Comment by remitamine** [comment]
now, it's working.
```
youtube-dl -v http://www.pbs.org/wgbh/nova/earth/earth-from-space.html
[debug] System config: []
[debug] User config: ['--external-downloader', 'aria2c', '--external-downloader-args', '-x 16 --check-certificate=false', '-f', 'best[height<=?720]', '--sub-lang', 'ar,en', '--write-sub']
[debug] Command-line args: ['-v', 'http://www.pbs.org/wgbh/nova/earth/earth-from-space.html']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.07.21
[debug] Python version 3.4.3 - Linux-4.1.2-2-ARCH-x86_64-with-arch-Arch-Linux
[debug] exe versions: ffmpeg 2.7.2, ffprobe 2.7.2, rtmpdump 2.4
[debug] Proxy map: {}
[PBS] earth-from-space: Downloading webpage
[PBS] earth-from-space: Downloading JSON metadata
[PBS] earth-from-space: Downloading recommended_encoding video url info
[PBS] earth-from-space: Downloading m3u8 information
[PBS] earth-from-space: Downloading alternate_encoding video url info
WARNING: ar subtitles not available for 2334144059
[info] Writing video subtitles to: Earth From Space-2334144059.en.ttml
[debug] Invoking downloader on 'http://ga.video.cdn.pbs.org/videos/nova/8e9b924e-56ec-4e53-b2c8-fc543054f771/30278/hd-mezzanine-16x9/NOVA4006_Summer2013_mezz16x9-16x9-hls-2500k.m3u8'
[download] Destination: Earth From Space-2334144059.mp4
ffmpeg version 2.7.2 Copyright (c) 2000-2015 the FFmpeg developers
  built with gcc 5.1.0 (GCC)
  configuration: --prefix=/usr --disable-debug --disable-static --disable-stripping --enable-avisynth --enable-avresample --enable-fontconfig --enable-gnutls --enable-gpl --enable-libass --enable-libbluray --enable-libfreetype --enable-libfribidi --enable-libgsm --enable-libmodplug --enable-libmp3lame --enable-libopencore_amrnb --enable-libopencore_amrwb --enable-libopenjpeg --enable-libopus --enable-libpulse --enable-libschroedinger --enable-libspeex --enable-libssh --enable-libtheora --enable-libv4l2 --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxvid --enable-shared --enable-version3 --enable-x11grab
  libavutil      54. 27.100 / 54. 27.100
  libavcodec     56. 41.100 / 56. 41.100
  libavformat    56. 36.100 / 56. 36.100
  libavdevice    56.  4.100 / 56.  4.100
  libavfilter     5. 16.101 /  5. 16.101
  libavresample   2.  1.  0 /  2.  1.  0
  libswscale      3.  1.101 /  3.  1.101
  libswresample   1.  2.100 /  1.  2.100
  libpostproc    53.  3.100 / 53.  3.100
Input #0, hls,applehttp, from 'http://ga.video.cdn.pbs.org/videos/nova/8e9b924e-56ec-4e53-b2c8-fc543054f771/30278/hd-mezzanine-16x9/NOVA4006_Summer2013_mezz16x9-16x9-hls-2500k.m3u8':
  Duration: 01:52:52.44, start: 10.000000, bitrate: 0 kb/s
  Program 0 
    Metadata:
      variant_bitrate : 0
    Stream #0:0: Video: h264 (Main) ([27][0][0][0] / 0x001B), yuv420p, 1280x720 [SAR 1:1 DAR 16:9], 29.97 fps, 29.97 tbr, 90k tbn, 59.94 tbc
    Stream #0:1: Audio: aac (LC) ([15][0][0][0] / 0x000F), 44100 Hz, stereo, fltp, 204 kb/s
[mp4 @ 0x55dabf19b1a0] Codec for stream 0 does not use global headers but container format requires global headers
[mp4 @ 0x55dabf19b1a0] Codec for stream 1 does not use global headers but container format requires global headers
Output #0, mp4, to 'Earth From Space-2334144059.mp4.part':
  Metadata:
    encoder         : Lavf56.36.100
    Stream #0:0: Video: h264 ([33][0][0][0] / 0x0021), yuv420p, 1280x720 [SAR 1:1 DAR 16:9], q=2-31, 29.97 fps, 29.97 tbr, 90k tbn, 90k tbc
    Stream #0:1: Audio: aac ([64][0][0][0] / 0x0040), 44100 Hz, stereo, 204 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (copy)
  Stream #0:1 -> #0:1 (copy)
Press [q] to stop, [?] for help
frame=   22 fps=0.0 q=-1.0 size=      88kB time=00:00:00.70 bitrate=1031.6kbits
```

**Comment by dstftw** [comment]
Closing since it works fine with the latest version.
